modules/acm/src/test/scala/edu/gemini/epics/acm/ObserveStateSpec.scala

```diff
@@ -785,12 +928,12 @@ final class ObserveStateSpec extends FunSuite with GsaoiMocks with NifsMocks wit
 class CadDirectiveChannelMock extends ReadWriteClientEpicsChannel[CadDirective] {
   override def setValue(a: java.util.List[CadDirective]): Unit = {}
   override def setValue(a : CadDirective): Unit = {}
-  override def getAll(): java.util.List[CadDirective] = null
-  override def getDBR(): gov.aps.jca.dbr.DBR = null
-  override def getFirst(): CadDirective = null.asInstanceOf[CadDirective]
-  override def getName(): String = null
-  override def getType(): gov.aps.jca.dbr.DBRType = null
-  override def isValid(): Boolean = true
```

Comment:
The IDE told me to remove the parentheses

Reply:
👍